apps/emqx/src/emqx_external_trace.erl

```diff
@@ -328,7 +370,17 @@ action_attrs(#{mod := Mod, func := Func}) ->
         'action.type' => <<"function">>,
         'action.function' => printable_function_name(Mod, Func),
         'client.clientid' => ?EXT_TRACE__ACTION_INTERNAL_CLIENTID
-    }.
+    };
+action_attrs(_) ->
+    ?fallback_attrs().
+
+fallback_attrs(FunctionName) ->
+    Msg = <<"Generate Attributes Failed">>,
+    ?SLOG(
```

Comment:
could this become a flood of error level logs ?

Reply:
NO
This is to avoid errors like function_caluse / case_caluse that cause the process to exit.